### PR TITLE
Clients controllers + schema

### PR DESCRIPTION
**Pull request by krummi**

### CODE REVIEW

File: web/views/layout_view.ex [hunk 1]
```diff
@@ -2,6 +2,7 @@ defmodule Appointments.LayoutView do
   use Appointments.Web, :view
 
   def active?(conn, path) do
-    if conn.request_path == "/#{path}", do: "active", else: ''
+    [_ | [resource | _]] = String.split(conn.request_path, "/")
```
Comment (kthelgason) [inline web/views/layout_view.ex]: could this possibly cause an unhandled error if `conn.request_path` has a different format than we expect, f.x. `"/"`

**Comment by kthelgason** [comment]
One comment, otherwise great stuff.

File: web/views/layout_view.ex (hunk 1 above)
Comment (krummi) [inline web/views/layout_view.ex]: In that case we would have something like this:
`[ a | [ b | c ]] = String.split("/", "/")` such that  `a=""`, `b=""` and `c=[]`.
But I guess would could make this prettier ;-)

**Comment by krummi** [comment]
I just updated the `active?` function

**Comment by kthelgason** [comment]
I'm on board with this ⛵

**Pull request merged**